ecosystem/sep-0038.md

```diff
@@ -30,7 +30,9 @@ An anchor must define the location of their `ANCHOR_QUOTE_SERVER` in their [`ste
 
 ### Authentication
 
-All endpoints require authentication in the form of a [SEP-10](sep-0010.md) JSON Web Token (JWT) in the `Authorization` header of the request. 
+This protocol's endpoints use authentication in the form of a [SEP-10](sep-0010.md) JSON Web Token (JWT) in the `Authorization` header of the request.
+
+While it is mandatory for [`POST /quote`](#post-quote) and [`GET /quote/:id`](#get-quote), it is optional for the remaining endpoints. When optional, the JWT token can be used to personalize the respective responses.
```

Comment:
Small wording suggestion, otherwise looks great.
```suggestion
Authentication is mandatory for [`POST /quote`](#post-quote) and [`GET /quote/:id`](#get-quote), but optional for the remaining endpoints. When optional, if authentication is provided it can be used to personalize the respective responses.
```

Reply:
https://github.com/stellar/stellar-protocol/pull/1144/commits/26a8534f440318987301ab0e3abcd08f36e8908c